tbtadm/controller.cpp

```diff
@@ -320,6 +331,9 @@ void tbtadm::Controller::run()
 
 void tbtadm::Controller::devices()
 {
+    if (!sysfsDeviceExists())
+        return;
+
```

Comment:
@ybernat I saw some code without braces for single statements, but other code has them, I personally like Linux kernel way more (then we need to convert other code to this style).